WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -313,10 +313,10 @@ import Foundation
             return "jetpack_disconnect_tapped"
         case .jetpackDisconnectRequested:
             return "jetpack_disconnect_requested"
-        case .jetpackWhitelistedIpsViewed:
-            return "jetpack_whitelisted_ips_viewed"
-        case .jetpackWhitelistedIpsChanged:
-            return "jetpack_whitelisted_ips_changed"
+        case .jetpackAllowlistedIpsViewed:
+            return "jetpack_allowlisted_ips_viewed"
+        case .jetpackAllowlistedIpsChanged:
+            return "jetpack_allowlisted_ips_changed"
```

Comment:
Not sure if this is okay to just change the event names, what could be the better/proper way of doing so @leandroalonso?

Reply:
Hi @ceyhun! Thanks for making those changes.

I think it's ok to change it for all user-facing changes. As for the analytic events this change needs to be made on both platforms (Android & iOS) to ensure consistency.

If that's the case, all good. :D

Reply:
It wasn't :D Just created a PR to update Android as well: https://github.com/wordpress-mobile/WordPress-Android/pull/14223